packages/arb-token-bridge-ui/src/components/common/SidePanel.tsx

```diff
@@ -1,50 +1,102 @@
+import { useState } from 'react'
 import { Dialog } from '@headlessui/react'
 import { twMerge } from 'tailwind-merge'
 import { XMarkIcon } from '@heroicons/react/24/outline'
-import { Transition } from './Transition'
+
+import { Transition } from '@headlessui/react'
 
 type SidePanelProps = {
   heading: string
   isOpen: boolean
   onClose?: () => void
   children: React.ReactNode
   panelClassNameOverrides?: string
+  scrollable?: boolean
 }
 
 export const SidePanel = ({
   isOpen,
   heading,
   onClose,
   children,
-  panelClassNameOverrides = ''
+  panelClassNameOverrides = '',
+  scrollable = true
```

Comment:
We don't want tx history side panel to be scrollable because the tx history itself has its own scrolling and we run into a double scroll issue.

We want other panels to be scrollable.